src/glyph.ts

```diff
@@ -7,34 +7,33 @@ import { Element } from './element';
 import { Font, FontGlyph } from './font';
 import { RenderContext } from './rendercontext';
 import { Stave } from './stave';
-import { Stem } from './stem';
 import { Tables } from './tables';
 import { Category } from './typeguard';
 import { defined, RuntimeError } from './util';
 
 export interface GlyphProps {
-  code_head: string;
-  dot_shiftY: number;
-  position: string;
-  rest: boolean;
-  line_below: number;
-  line_above: number;
-  stem_beam_extension: number;
-  stem_up_extension: number;
-  stem_down_extension: number;
-  stem: Stem;
-  code: string;
-  code_flag_upstem: string;
-  code_flag_downstem: string;
-  flag: boolean;
-  width: number;
-  text: string;
-  tabnote_stem_down_extension: number;
-  tabnote_stem_up_extension: number;
-  beam_count: number;
-  shift_y: number;
-  getWidth(a?: number): number;
-  getMetrics(): GlyphMetrics;
+  code_head?: string;
+  leger_code_head?: string;
```

Comment:
it's `ledger` by the way, not `ledger. like `ledger note`.

Reply:
I also thought about this one, and I found:

https://en.m.wikipedia.org/wiki/Ledger_line

Perhaps we should unify.

Reply:
@sschmidTU Beginner music books in the US use ledger.  So Hal Leonard does not agree :)  But I went with leger b/c that's what Smufl uses, and I feel like we're a Smufl shop.

Reply:
I prefer using `ledger` in this codebase (mainly to be consistent.)

Reply:
Ok I understand everything ledger except for the Smufl glyph IDs.

Reply:
Done